cmd/gapir/cc/main.cpp

```diff
@@ -395,8 +536,134 @@ void android_process(struct android_app* app, int32_t cmd) {
 
 }  // namespace
 
+// Extract command line arguments from the extra of Android intent:
+//   adb shell am start -n <...> -e gapir-launch-args "'list of arguments to be
+//   extracted'"
+// Note the quoting: from host terminal adb command, we need to double-escape
```

Comment:
We have run into some insane quoting things on windows so we should test there as well.

Reply:
This is copied from Graphics Fuzz.

Reply:
Alrighty. We should just make double-sure to test on windows before the next release.